CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -388,123 +396,40 @@ class CameraFragment : Fragment() {
                     // Display flash animation to indicate that photo was captured
                     container.postDelayed({
                         container.foreground = ColorDrawable(Color.WHITE)
-                        container.postDelayed(
-                                { container.foreground = null }, ANIMATION_FAST_MILLIS)
+                        container.postDelayed({container.foreground = null}, ANIMATION_FAST_MILLIS)
                     }, ANIMATION_SLOW_MILLIS)
                 }
             }
         }
 
         // Listener for button used to switch cameras
         controls.findViewById<ImageButton>(R.id.camera_switch_button).setOnClickListener {
-            lensFacing = if (CameraX.LensFacing.FRONT == lensFacing) {
-                CameraX.LensFacing.BACK
+            lensFacing = if (CameraSelector.LENS_FACING_FRONT == lensFacing) {
+                CameraSelector.LENS_FACING_BACK
             } else {
-                CameraX.LensFacing.FRONT
+                CameraSelector.LENS_FACING_FRONT
             }
-            try {
-                // Only bind use cases if we can query a camera with this orientation
-                CameraX.getCameraWithLensFacing(lensFacing)
-
-                // Unbind all use cases and bind them again with the new lens facing configuration
-                CameraX.unbindAll()
-                bindCameraUseCases()
-            } catch (exc: Exception) {
-                // Do nothing
-            }
-        }
-
-        // Listener for button used to view last photo
-        controls.findViewById<ImageButton>(R.id.photo_view_button).setOnClickListener {
-            Navigation.findNavController(requireActivity(), R.id.fragment_container).navigate(
-                    CameraFragmentDirections.actionCameraToGallery(outputDirectory.absolutePath))
-        }
-    }
-
-
-    /**
-     * Our custom image analysis class.
-     *
-     * <p>All we need to do is override the function `analyze` with our desired operations. Here,
-     * we compute the average luminosity of the image by looking at the Y plane of the YUV frame.
-     */
-    private class LuminosityAnalyzer(listener: LumaListener? = null) : ImageAnalysis.Analyzer {
```

Comment:
Why are you removing this? Please revert, it is unrelated to updating to `alpha08` and makes reviewing this PR much harder.

Reply:
It is the same as it was as an inner class. Assuming one might want to add more than just one type of `ImageAnalysis.Analyzer`, it makes it easier to extend - and this code has no relation to a `Fragment` (UI) at all.

Reply:
Since the current analyzer is very small and the file size is not yet too big, I think it's better to keep things condensed in less files. Also, this is unrelated to `alpha08` which makes it harder for us to evaluate this PR. If you want to do this, I'd suggest adding it as part of another PR so it can be discussed separately -- but for now I don't think it's the right approach.